docs/mint.json

```diff
@@ -311,7 +311,8 @@
           "group": "Guides",
           "pages": [
             "self-hosting/guides/mongo-to-postgres",
-            "self-hosting/guides/custom-certificates"
+            "self-hosting/guides/custom-certificates",
+            "self-hosting/guides/automated-bootstrapping"
```

Comment:
comment: i wonder if this will give it enough visibility. seems like something people might want easier reach to. maybe we can call the title programmatic provisioning? this why it catches people's attention more?